### PR TITLE
FPGA/Computing Platform Reqs

### PR DESCRIPTION
**Pull request by peterdeutsch**
These are the requirements for the FPGA/CP _itself_ - subrequirements (i.e. for the camera, ML implementation, etc) are not included.

### CODE REVIEW

File: docs/reqs/reqs.tex [hunk 1]
```diff
@@ -211,8 +218,9 @@ \subsection{Non-RPAS Air-Ground I/O}
 
 \subsection{Computing Platform}
 \begin{enumerate}[{F.CP}.1:]
-	\item Captures video at a minimum resolution of 640x480 at 10fps.
-	\item Onboard FPGA is capable of performing parallel matrix computation to speedup machine learning.
```
Comment (FSXAC) [inline docs/reqs/reqs.tex]: These are non function requirements. The high level requirements would fit well here though (I meant to select line 221-223)

File: docs/reqs/reqs.tex [hunk 2]
```diff
@@ -240,6 +252,12 @@ \subsection{Multirotor RPAS}
     \item The drone should have systems for low-battery alarm and emergency protocol for low-battery and loss of control radio.
 \end{enumerate}
 
+\subsection{Computing Platform}
+\begin{enumerate}[{NF.CP}.1:]
+	\item Computing Platform must be capable of performing a power-on self-test (POST) to ensure communications and video links are established
```
Comment (FSXAC) [inline docs/reqs/reqs.tex]:
```suggestion
	\item Computing Platform must perform a power-on self-test (POST) to ensure communications and video links are established
```

File: docs/reqs/reqs.tex [hunk 3]
```diff
@@ -240,6 +252,12 @@ \subsection{Multirotor RPAS}
...
+	\item Computing Platform must keep its configuration across flight sessions, including if the battery has been detached
```
Comment (FSXAC) [inline docs/reqs/reqs.tex]: Suggest:

...must have persistent configuration using non-volatile memory even in the case of power cut

File: docs/reqs/reqs.tex [hunk 4]
```diff
@@ -259,7 +277,10 @@ \subsection{Cost}
 
 \subsection{Platform Extensibility}
 \begin{enumerate}[{C.PE}.1:]
-    \item Significant on-chip area (>50\%) should be dedicated for future client use.
+    \item FPGA must contain at least 60,000 logic elements to provide sufficient on-chip capacity for initial and future machine learning implementations.
+    \item No more than 10,000 logic elements may be reserved for glue logic.
```
Comment (FSXAC) [inline docs/reqs/reqs.tex]: Suggest:

Glue logic must not reserve more than 10000 logic elements

File: docs/reqs/reqs.tex [hunk 5]
```diff
@@ -184,17 +184,24 @@ \section{Background}
...
+This section outlines the functional requirements of the integrated system.
 
-Functional requirements describes what the project do. It embodies the essential requirements that must be completed in order for the project to be deemed deliverable or successful. The functional requirements include the scope of business rules, legal or regulartory requirements, certification requirements, and external interfacing requirements for the project.
+Functional requirements describes what the project do. It embodies the essential requirements that must be completed in order for the project to be deemed deliverable or successful. The functional requirements include the scope of business rules, legal or regulatory requirements, certification requirements, and external interfacing requirements for the project.
+
+\subsection{High-Level Requirements}
+\begin{enumerate}[{NF.HL}.1:]
+    \item Computing Platform must perform hardware-accelerated machine-learning.
+    \item Computing Platform must be mounted on a drone capable of sustained flight.
+    \item Machine Learning Implementation must implement a novel video processing use-case, such as object detection.
```
Comment (FSXAC) [inline docs/reqs/reqs.tex]: I don’t think machine learning implementation should be capitalized

File: docs/reqs/reqs.tex [hunk 6]
```diff
@@ -160,7 +160,7 @@
 % Begin content
 \section{About This Document}\label{section:about}
 \subsection{Purpose}\label{section:about:purpose}
-This document outlines the project's requirements, laying out a basis of what is required for the project to be deemed \textit{successful}. In particular, this document outlines expected metrics related to performance, quality/longevity, environmental impact, cost, legality, and safety.
+This document outlines the project's requirements, laying out a basis of what is required for the project to be deemed \textit{successful}. In particular, this document outlines expected metrics related to performance, quality, longevity, environmental impact, cost, legality, and safety.
```
[the remark this answers is not in the document]
Comment (peterdeutsch) [reply docs/reqs/reqs.tex]: Agreed

**Pull request merged**